overlord/configstate/hooks.go

```diff
@@ -96,7 +96,8 @@ func (h *configureHandler) Before() error {
 		if err != nil && err != state.ErrNoState {
 			return err
 		}
-		if len(patch) != 0 {
+		// core does not need a configure hook
```

Comment:
// core is handled internally and does not need a configure hook

Reply:
Missed that when I was pushing the updates. I'll fix this once the PR lands on master.